common/workunit/workunit.cpp

```diff
@@ -6382,9 +6381,9 @@ IWUResult* CLocalWorkUnit::updateTemporaryByName(const char *qname)
     if (existing)
         return (IWUResult *) existing;
     if (!temporaries.length())
-        p->addPropTree("Temporaries", createPTree("Temporaries"));
+        p->addPropTree("Temporaries");
```

Comment:
trivial not new, would be slightly better (maybe) if these 3 lines were:  (to avoid the add and query)
```
IPropertyTree *vars = (temporaries.length()) ? p->queryPropTree("Temporaries") : p->addPropTree("Temporaries");
```


.. or jptree method to do same, as quite common.